pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -500,6 +500,16 @@ def make_package(args):
         if exists('build.properties'):
             os.remove('build.properties')
 
+    # Apply java source patches if any are present:
+    if exists('src/patches'):
+        print("Applying Java source code patches...")
+        for patch_name in os.listdir('src/patches'):
+            patch_path = 'src/patches/' + patch_name
```

Comment:
Minor: we usually make path not OS specific, e.g. using `os.path.join` or `os.path.sep`